packages/tendermint-rpc/src/tendermint34/adaptor/index.ts

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable @typescript-eslint/naming-convention */
-import { Adaptor } from "../../adaptor";
-import { hashBlock, hashTx } from "../../hasher";
+import { hashBlock, hashTx } from "../hasher";
 import { Params } from "./requests";
 import { Responses } from "./responses";
+import { Adaptor } from "./types";
 
-export const v0_33: Adaptor = {
+export { Decoder, Encoder, Params, Responses } from "./types";
+
+export const adaptor34: Adaptor = {
```

Comment:
`/* eslint-disable @typescript-eslint/naming-convention */` can probably be removed